container/attributes.go

```diff
@@ -12,7 +12,66 @@ const (
 
 	// SysAttributeZone is a string of zone for container name.
 	SysAttributeZone = SysAttributePrefix + "ZONE"
+
+	// SysAttributeHomomorphicHashing is a container's homomorphic hashing state.
+	SysAttributeHomomorphicHashing = SysAttributePrefix + "HOMOMORPHIC-HASHING"
```

Comment:
We commonly use underscore as a word delimiter in well-known attributes. See object for examples.